apps/liquidity-provision-dashboard/src/app/components/dashboard/market-list/market-list.tsx

```diff
@@ -96,6 +97,48 @@ export const MarketList = () => {
             headerTooltip={t('The market name and settlement asset')}
           />
 
+          <AgGridColumn
+            headerName={t('Market Code')}
+            headerTooltip={t(
+              'The market code is a unique identifier for this market'
+            )}
+            field="tradableInstrument.instrument.code"
+          />
+
+          <AgGridColumn
+            headerName={t('Type')}
+            headerTooltip={t('Type')}
+            field="tradableInstrument.instrument.product.__typename"
+          />
+
+          <AgGridColumn
+            headerName={t('Last Price')}
+            headerTooltip={t('Latest price for this market')}
+            field="data.markPrice"
+            valueFormatter={({ value, data }: ValueFormatterParams) =>
```

Comment:
suggestion (non blocking): There's a type helper `VegaValueFormatterParams` you can use to get value/data typed properly

Reply:
Oh great, thanks! I was wondering if there was a way to do that (: 

I tried to apply it to some of these fields and I think it's (likely correctly) challenging some assumptions that are there. If it's alright with you I'll follow up with this in another PR?